src/frontend/src/mock/setup-mock.ts

```diff
@@ -4,13 +4,15 @@ import { veraDeploys } from './deploys';
 import { apiBaseUri } from '../utils/config';
 import { getMockIssue } from './jira-issue-mock';
 import { getMockCommits } from './commit-for-release-mock';
+import { teams } from './teams';
 
 export function setupMock() {
     /* tslint:disable-next-line */
     console.log('### MOCK ENABLED! ###');
     (fetchMock as any)._mock();
 
-    fetchMock.get(apiBaseUri + '/deploy', respondWith(delayed(500, veraDeploys)));
+    fetchMock.get(apiBaseUri + `/deploy?team=${teams[0].id}`, respondWith(delayed(500, veraDeploys)));
```

Comment:
tenker det hadde vært veldig kult om disse 2 returnerte forskjellige apper, slik at mock-versjonen er mer representativ for virkeligheten.